src/test.ts

```diff
@@ -15,7 +15,9 @@ getTestBed().initTestEnvironment(
   BrowserDynamicTestingModule,
   platformBrowserDynamicTesting()
 );
+
+//TODO: move the package into a project and change the context
```

Comment:
Please fix errors from `yarn lint` for CI to pass: https://travis-ci.org/FortAwesome/angular-fontawesome/builds/583252496#L245

You can ignore warnings output by the command.